snapshotter/cmd/devmapper/flag.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"strings"
+)
+
+// parseKeyValueOpt splits a string to key=value
+func parseKeyValueOpt(opt string) (string, string, error) {
+	pair := strings.SplitN(opt, "=", 2)
```

Comment:
`strings.SplitN("=", "=", 20)` will return `[ "", "" ]`. Is that correct behavior?
`strings.SplitN("=bar")` will return `["", "bar"]`
`strings.SplitN("bar=")` will return `["bar", ""]`.

Reply:
Technically empty key would break anything, it just wouldn't much any key we expect. But I've added an extra error check.